plugins/node/opentelemetry-instrumentation-net/src/net.ts

```diff
@@ -92,7 +92,7 @@ export class NetInstrumentation extends InstrumentationBase<Net> {
   /* It might still be useful to pick up errors due to invalid connect arguments. */
   private _startGenericSpan(socket: Socket) {
     const span = this.tracer.startSpan('connect', {
-      kind: SpanKind.CLIENT,
+      kind: SpanKind.INTERNAL,
```

Comment:
`SpanKind.INTERNAL` is the default so you could omit the second argument.

Reply:
Thanks, good point :slightly_smiling_face:  Fixed it